test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateBuildOptions.cs

```diff
@@ -84,6 +84,36 @@ public void MigratingWebProjectWithoutCustomSourcesOrResourcesDoesNotEmitCompile
             mockProj.Items.Count().Should().Be(0);
         }
 
+        public void MigratingOutputNamePopulatesAssemblyName()
+        {
+            var mockProj = RunBuildOptionsRuleOnPj(@"
+                {
+                    ""buildOptions"": {
+                        ""outputName"": ""some name""
+                    }
+                }");
+
+            mockProj.Properties.Count(p => p.Name == "AssemblyName").Should().Be(1);
+            mockProj.Properties.First(p => p.Name == "AssemblyName").Value.Should().Be("some name");
+        }
+
+        [Fact]
+        public void MigratingOutputNamePopulatesPackageIdWithTheProjectContainingFolderName()
+        {
+            var testDirectoryPath = Temp.CreateDirectory().Path;
```

Comment:
maybe not for this PR, but this looks like a Unit test that works in memory... why do we need to create the directory?